packages/prettyhtml-formatter/test/fixtures/angular/output.html

```diff
@@ -10,6 +10,7 @@
     [style.paddingBottom.%]="50 * cellAspectRatio / numCols"
   >{{label}}</td>
 </tr>
+
```

Comment:
Please add a separate fixture for the gap feature

Reply:
Existing fixtures with gaps still need to be modified right? Since there is no option to deactivate the feature

Reply:
Yes, just add another fixture to ensure that we won't forget the behaviour.